server/src/services/cart.service.ts

```diff
@@ -0,0 +1,21 @@
+import CartError, { CartErrorType } from '../errors/cart.error';
+import { ICartData } from '../interfaces/cartService.interface';
+import Cart from '../models/cart.model';
+import ItemOption from '../models/itemOption.model';
+
+class CartService {
+  private async _checkOptionValid(item_id: number, option_id: number) {
+    const targetOption = await ItemOption.findByPk(option_id, { raw: true });
+    if (targetOption?.item_id !== item_id) {
+      throw new CartError(CartErrorType.NOT_VALID_OPTION_ID);
+    }
+  }
+  public async createCart(cart: ICartData): Promise<Cart> {
+    // validation: 아이템이 가지고 있는 item_option인지 검사하기
+    await this._checkOptionValid(cart.item_id, cart.option_id);
+    const newCart = (await Cart.create(cart)).get();
+    return newCart;
```

Comment:
```suggestion
    return Cart.create(cart);
```
이렇게만 해줘도 같게 넘어가긴 합니당! 👀